cpp/dolfinx/mesh/graphbuild.cpp

```diff
@@ -500,6 +502,9 @@ graph::AdjacencyList<std::int64_t> compute_nonlocal_dual_graph(
                                   std::next(data.begin(), offsets[node + 1]));
       std::ranges::sort(links);
       auto duplicate_links = std::ranges::unique(links);
+      if (duplicate_links.size() == 0)
```

Comment:
Could this be changed to `duplicate_links.empty()`? Or does it not have that option?